server/app/tool/src/attestation.rs

```diff
@@ -0,0 +1,60 @@
+use anyhow::{anyhow, Context, Result};
+use serde_json::json;
+use sha2::{Digest, Sha384};
+use std::fs;
+
+use crate::attestation_api::attestation_service_client::AttestationServiceClient;
+use crate::attestation_api::{AttestationRequest, AttestationResponse};
+
+const ATTESTATION_SOCK: &str = "127.0.0.1:3000";
+
+fn evidence() -> Result<String> {
```

Comment:
1. rename to `sample_evidence()` or something like that. 
2. As I said above, do we need this as part of the application? Maybe it's enough as part of the unit/integration tests?
3. curious: why is this `Result<String>` and not simply `String` ?